package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "azure-sb",
+  "version": "0.11.0",
+  "description": "Azure ServiceBus SDK for Node.js",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "./dist/lib/index.js",
+  "types": "./typings/lib/index.d.ts",
+  "dependencies": {
+    "async-lock": "^1.1.2",
+    "debug": "^3.1.0",
+    "is-buffer": "2.0.2",
+    "ms-rest": "^2.3.3",
+    "ms-rest-azure": "^2.5.5",
+    "rhea": "^0.2.14",
+    "tslib": "^1.9.2",
+    "uuid": "^3.2.1"
+  },
+  "devDependencies": {
+    "@types/uuid": "^3.4.3",
+    "@types/async-lock": "^1.1.0",
+    "@types/debug": "^0.0.30",
+    "@types/node": "^8.0.37",
+    "@types/chai": "^4.1.2",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/mocha": "^5.0.0",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "mocha": "^5.0.5",
+    "ts-node": "^5.0.1",
+    "tslint": "^5.9.1",
+    "typescript": "^2.9.1",
+    "dotenv": "^5.0.1",
+    "@types/dotenv": "^4.0.3",
```

Comment:
nit: let npm modify the dependency list for you through `npm i <package>` (only reason i can tell is because it automatically sorts all dependencies each time)